compiler/src/iree/compiler/Dialect/Flow/Transforms/test/dispatch_linalg_on_tensors_fusion_reduction_broadcast_elementwise.mlir

```diff
@@ -0,0 +1,33 @@
+// RUN: iree-opt --iree-flow-fuse-reduction-broadcast-elementwise --split-input-file --verify-diagnostics --pass-pipeline="func.func(iree-flow-dispatch-linalg-on-tensors-pass)" --canonicalize -cse %s | FileCheck %s
```

Comment:
We don't need `verify-diagnostics`, right? We won't verify if there are diagnostics in the pass.

Reply:
Done.